src/sentry/api/serializers/models/team.py

```diff
@@ -187,20 +197,43 @@ def serialize(
         return d
 
 
+def get_scim_teams_members(
+    team_list: Sequence[Team],
+) -> MutableMapping[Team, MutableSequence[MutableMapping[str, Any]]]:
+    members = RangeQuerySetWrapper(
+        OrganizationMember.objects.filter(teams__in=team_list).select_related("user"),
+        limit=10000,
+    )
+    member_map: MutableMapping[Team, MutableSequence[MutableMapping[str, Any]]] = defaultdict(list)
+    for member in members:
+        for team in member.teams.all():
+            member_map[team].append({"value": str(member.id), "display": member.get_email()})
+    return member_map
+
+
 class TeamSCIMSerializer(Serializer):  # type: ignore
+    def get_attrs(
+        self, item_list: Sequence[Team], user: Any, **kwargs: Any
+    ) -> MutableMapping[Team, MutableMapping[str, Any]]:
+        result: MutableMapping[Team, MutableMapping[str, Any]] = {}
+        for team in item_list:
+            result[team] = {"members": None}
+
+        # For a patch request, we don't need to return the full list of members
+        # so look at exclude_members arg
+        if not kwargs.get("exclude_members", None):
```

Comment:
I'd suggest using our `expand/collapse` pattern rather than custom keywords. By default I'd recommend a serializer return as little info as possible, so we could make this an include param like `'members'` instead.

https://github.com/getsentry/sentry/blob/ddd17ea3c81287a3c6a49d51540822530a468875/src/sentry/api/serializers/models/projectcodeowners.py#L11 as an example, although you'd just be using it in `get_attrs` instead.

Reply:
👍🏼